torch_ttnn/passes/to_tt_pass.py

```diff
@@ -368,6 +368,17 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                     new_node,
                     delete_user_cb=lambda node: node != new_node,
                 )
+            if node.target == torch.ops.aten.transpose.int:
```

Comment:
what `int` stands for in `transpose.int`?

Reply:
Integer inputs. There could be other overloaded variants of transpose. This one is for integers.

Reply:
how hard is it to handle non-integer transposes? why don't we just cover all variants here?

Reply:
It can be covered, have to check how many overloads are there and include them. I will capture this in the next commit.

Reply:
@ayerofieiev-tt I had a look at transpose overloads. There are two of them:
**torch.ops.aten.transpose.int** (uses integer indices to specify the dimensions to be transposed)
**torch.ops.aten.transpose.Dimname** (uses dimension names to specify the dimensions to be transposed)
Dimname is used for tensor dimension having names rather than integer indices. So int overload pretty much covers all input datatypes.